utils/metrics.py

```diff
@@ -49,15 +49,15 @@ def average(self):
         return self.avg
 
 
-def report_classification(pred, label, batch_size, metrics_dict):
+def report_classification(pred, label, batch_size, metrics_dict, ignore_index=-1):
     """Computes precision, recall and f-score for each class and average of all classes.
     http://scikit-learn.org/stable/modules/generated/sklearn.metrics.classification_report.html
     """
     class_report = classification_report(label.cpu(), pred.cpu(), output_dict=True)
 
     class_score = {}
     for key, value in class_report.items():
-        if key not in ['micro avg', 'macro avg', 'weighted avg', 'accuracy']:
+        if key not in ['micro avg', 'macro avg', 'weighted avg', 'accuracy'] and key != str(ignore_index):
```

Comment:
Ignore metrics from class_report.items() if key is ignore_index. Has been tested.